lighthouse-core/audits/dobetterweb/uses-http2.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Audit a page to ensure that resource loaded over its own
+ * origin are over the http/2 protocol.
+ */
+
+'use strict';
+
+const Audit = require('../audit');
+const Formatter = require('../../formatters/formatter');
+
+class UsesHTTP2Audit extends Audit {
+
+  /**
+   * @return {!AuditMeta}
+   */
+  static get meta() {
+    return {
+      category: 'Performance',
+      name: 'uses-http2',
+      description: 'Site resources use HTTP/2',
+      helpText: 'See <a href="https://http2.github.io/faq/" target="_blank">HTTP2 FAQ</a>.',
+      requiredArtifacts: ['SameOriginResources']
+    };
+  }
+
+  /**
+   * @param {!Artifacts} artifacts
+   * @return {!AuditResult}
+   */
+  static audit(artifacts) {
+    if (typeof artifacts.SameOriginResources === 'undefined' ||
+        !Array.isArray(artifacts.SameOriginResources)) {
+      return UsesHTTP2Audit.generateAuditResult({
+        rawValue: -1,
+        debugString: 'SameOriginResources gatherer did not run'
+      });
+    }
+
+    // Filter the non http/2 resources.
+    const resources = artifacts.SameOriginResources.filter(record => {
+      return /HTTP\/[01][\.\d]?/i.test(record.protocol);
```

Comment:
I don't think this field is the header, but the string identifier internal to devtools (which also looks like what's shown in the network panel). From a quick look at a few sites it's giving me `'http/1.1'` for requests from e.g. example.com and `'h2'` for requests from e.g. chromestatus